openassessment/xblock/static/sass/oa/utilities/_shame.scss

```diff
@@ -4,6 +4,7 @@
 // NOTES: use for any bad-form/orphaned scss that knowingly violate edX FED architecture/standards (see - http://csswizardry.com/2013/04/shame-css)
 
 .openassessment {
+  @include direction();
```

Comment:
I moved it from `_shame.scss`, now I know why there's shame :+1: 

And for the mixin, the one I'm using is this: https://github.com/edx/edx-ora2/blob/master/openassessment/xblock/static/sass/vendor/bi-app/_mixins.scss#L266

and it's just a simple `@include direction();` and nothing more.